lua/cmp_nvim_lsp/init.lua

```diff
@@ -50,7 +50,9 @@ end
 
 ---Backwards compatibility
 M.update_capabilities = function(capabilities, override)
-  vim.deprecate('cmp_nvim_lsp.update_capabilities', 'cmp_nvim_lsp.default_capabilities', '1.0.0', 'cmp-nvim-lsp')
+  if vim.deprecate then
+    vim.deprecate('cmp_nvim_lsp.update_capabilities', 'cmp_nvim_lsp.default_capabilities', '1.0.0', 'cmp-nvim-lsp')
+  end
```

Comment:
Please add print message.

I think the user should follow the latest configuration.

Reply:
I backported `vim.deprecate` for neovim < 0.9.